rustc 1.52.1 issue

After the last round of `pacman` updates, `rustc.exe` no longer works for me on a fully updated Windows 10 pro install:

```
victor@win10pro64 MINGW64 ~
$ rustc
C:/msys64/mingw64/bin/rustc.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory

victor@win10pro64 MINGW64 ~
$ pacman -Q|grep rust
mingw-w64-i686-rust 1.52.1-1
mingw-w64-x86_64-rust 1.52.1-1
```

We also see this in our github-ci setup here:
https://github.com/OISF/suricata/pull/6129/checks?check_run_id=2622070380 (workflow is here https://github.com/OISF/suricata/blob/master/.github/workflows/builds.yml#L1369)
```
2021-05-19T16:06:24.4338900Z ##[group]Run cargo install --root /usr --force --debug --version 0.14.1 cbindgen
2021-05-19T16:06:24.4339872Z [36;1mcargo install --root /usr --force --debug --version 0.14.1 cbindgen[0m
2021-05-19T16:06:24.4347975Z shell: D:\a\_temp\msys\msys2.CMD {0}
2021-05-19T16:06:24.4348423Z env:
2021-05-19T16:06:24.4349001Z   DEFAULT_LIBHTP_REPO: https://github.com/OISF/libhtp
2021-05-19T16:06:24.4349642Z   DEFAULT_LIBHTP_BRANCH: 0.5.x
2021-05-19T16:06:24.4350115Z   DEFAULT_LIBHTP_PR: 
2021-05-19T16:06:24.4350875Z   DEFAULT_SU_REPO: https://github.com/OISF/suricata-update
2021-05-19T16:06:24.4351481Z   DEFAULT_SU_BRANCH: master
2021-05-19T16:06:24.4351894Z   DEFAULT_SU_PR: 
2021-05-19T16:06:24.4352613Z   DEFAULT_SV_REPO: https://github.com/OISF/suricata-verify
2021-05-19T16:06:24.4353217Z   DEFAULT_SV_BRANCH: master
2021-05-19T16:06:24.4353631Z   DEFAULT_SV_PR: 
2021-05-19T16:06:24.4354343Z   DEFAULT_CFLAGS: -Wall -Wextra -Werror -Wno-unused-parameter -Wno-unused-function
2021-05-19T16:06:24.4355207Z   DEBIAN_FRONTEND: noninteractive
2021-05-19T16:06:24.4355677Z   RUST_VERSION_KNOWN: 1.49.0
2021-05-19T16:06:24.4356085Z   RUST_VERSION_MIN: 1.41.1
2021-05-19T16:06:24.4356489Z   MSYSTEM: MINGW64
2021-05-19T16:06:24.4356861Z ##[endgroup]
2021-05-19T16:06:24.7663176Z     Updating crates.io index
2021-05-19T16:06:43.1514950Z  Downloading crates ...
2021-05-19T16:06:43.3655557Z   Downloaded cbindgen v0.14.1
2021-05-19T16:06:44.1462433Z error: process didn't exit successfully: `rustc -vV` (exit code: 0xc0000135, STATUS_DLL_NOT_FOUND)
2021-05-19T16:06:44.1598670Z ##[error]Process completed with exit code 101.
```
Please let me know if I can provide additional info to help investigate the issue.

--- a/mingw-w64-rust/PKGBUILD
+++ b/mingw-w64-rust/PKGBUILD
@@ -8,7 +8,7 @@ pkgbase=mingw-w64-${_realname}
 pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}"
          "${MINGW_PACKAGE_PREFIX}-rust-docs")
 pkgver=1.52.1
-pkgrel=1
+pkgrel=2
 pkgdesc="Systems programming language focused on safety, speed and concurrency (mingw-w64)"
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64')
@@ -117,6 +117,7 @@ check() {
 package_rust() {
   depends=("${MINGW_PACKAGE_PREFIX}-gcc"
            "${MINGW_PACKAGE_PREFIX}-curl"
+           "${MINGW_PACKAGE_PREFIX}-libxml2"
            "${MINGW_PACKAGE_PREFIX}-libssh2")
 
   cd "${srcdir}/${CARCH}"

